packages/manager/modules/sms/src/sms/batches/routing.js

```diff
@@ -0,0 +1,15 @@
+import component from './telecom-sms-batches.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('sms.service.batches', {
+    views: {
+      'smsInnerView@sms.service': {
+        component: component.name,
+      },
+    },
+    translations: {
```

Comment:
I don't know if this should be replace to prefer using `@ngTranslationsInject` annotation instead

Reply:
Hmm yes I had the same question. Maybe @cbourgois or @antleblanc could tell anything on this?

Reply:
Yes, IMHO we should prefer `@ngTranslationsInject` instead of state translations. 
Here the component is exposed and use translations, but translations are only loaded when we reach the state url. 
If we want to reuse the component in another state, translations won't be loaded.

If translations are injected with `@ngTranslationsInject`, translations are available when the module is loaded.

Reply:
Ok, so the use of `@ngTranslationsInject` requires `angular-translate` and its own dependencies.
The whole `sms` module doesn't have this dependencies.
As said here https://github.com/ovh/manager/pull/3643#discussion_r490780712.

I propose to tackle down this changes in another PR, in order to make this changes in every module of `sms` section.
I'll create a JIRA ticket to do this, as it will be required before migrating the section to a ufrontend app.

Reply:
@JeremyDec But the `translations` decorator also does. If I have a look to the rollup plugin handling it, I see that we inject  `$translate` https://github.com/ovh/manager/blob/master/packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts#L94 so I'm having trouble to understand why it is technically impossible in that case (apart that dependency declaration is not correct but it's already the case)

Reply:
Hmm ok I've just test it now it is working... don't know why it hasn't work at first time. Will make the changes so